examples/oci/ingress.yaml

```diff
@@ -15,8 +15,8 @@ spec:
                 name: jaeger-query
                 port:
                   number: 16686
-          - path: /grafana
-            pathType: Prefix
+          - path: /grafana(/|$)(.*)
```

Comment:
`(/|$)` doesn't make sense if followed by `.*`, since `$` marks the end of the string